src/App.js

```diff
@@ -1,24 +1,16 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from 'react';
+import DataPage from './Pages/DataPage'
+import LoginPage from './Pages/LoginPage';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+  
+  const [login, setLogin] = useState(false)
+
+  let loginHandler = () =>{
+    setLogin(!login)
```

Comment:
You need to handle route change in `LoginPage`

Reply:
done